src/cljs/playground_coffeeshop/handlers.cljs

```diff
@@ -129,7 +129,8 @@
 
           menu-items (filter-items = "menu" response-items)
 
-          consignment-items (filter-items = "consignment" response-items)
+          consignment-item (-> (filter-items = "consignment" response-items)
```

Comment:
i like the change to singular, makes more sense, we should change `about-items` too since there's only one of those.